CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Eclipse GLSP Server Node Changelog
 
+## v1.1.0 - upcoming
+
+### Breaking Changes
+
+-   [graph] Align GGraph model with newest changes from glsp-server [#22](https://github.com/eclipse-glsp/glsp-server-node/pull/22) - Contributed on behalf of STMicroelectronics
+    -   Renamed interfaces:
+        -   `EdgePlacement` -> `GEdgePlacement` (affected classes: `GEdgeLayoutable`, `GLabel`)
+        -   `GLayoutContainer` (`glayout-container.ts`) -> `GLayouting` (`glayouting.ts`) (affected classes: `GCompartment`, `GGraph`, `GNode`)
```

Comment:
You can omit the file information here ('glayout-container.ts`) etc. GLayoutContainer is default exported so a source file name changes does not break anything for adopters.